lms/static/scripts/frontend_apps/components/JSTORPicker.js

```diff
@@ -211,6 +217,36 @@ export default function JSTORPicker({ onCancel, onSelectURL }) {
             </div>
           )}
 
+          {metadata.data && (
+            <div className="grow flex flex-col justify-end">
+              <div className="flex space-x-2 items-center px-1">
+                <div className="grow text-right">
+                  <label htmlFor="accept-jstor-terms">
+                    I accept the{' '}
+                    <Link href="https://about.jstor.org/terms/" target="_blank">
+                      terms and conditions
```

Comment:
Have we got an underlined link variant? Given that "terms and conditions" appears in the middle of a sentence, I think it might be wise for a11y to provide a stronger indicator that it is a link.

Reply:
I can definitely make this so with styling.

As future stuff:

* Updates of the `Link` shared component to take an `underline` prop is already in the works
* Further ahead: campaigning to make the default link styling underlined (we have a strong legacy of not underlining our links)